service/src/test/resources/application-test.yml

```diff
@@ -57,14 +57,6 @@ landingzone:
       southcentralus: /subscriptions/c5f8eca3-f512-48cb-b01f-f19f1af9014c/resourceGroups/terra-dev-protected-data-logs-rg/providers/Microsoft.Storage/storageAccounts/ltslogsdevsthcntrlus
     sentinelScheduledAlertRuleTemplateIds:
       - 0b9ae89d-8cad-461c-808f-0494f70ad5c4
-      - 8ee967a2-a645-4832-85f4-72b635bcb3a6
```

Comment:
Removing these to mitigate some of the flakiness as I want the IT pipeline stable. Ticketed here to [reintroduce](https://broadworkbench.atlassian.net/browse/WOR-1333
).